runtime/etc/Makefile.mli-shared

```diff
@@ -21,6 +21,15 @@
 
 include $(CHPL_MAKE_HOME)/runtime/etc/Makefile.include
 
+#
+# Taken from $CHPL_HOME/third-party/gmp/Makefile
+#
+HOST_CC=$(shell CHPL_MAKE_HOME=$(CHPL_MAKE_HOME) CHPL_MAKE_HOST_TARGET=--host make -f $(CHPL_MAKE_HOME)/make/Makefile.base printQ-CC)
+TARG_CC=$(CC)
+
+CLIENT_CC=$(HOST_CC)
+SERVER_CC=$(TARG_CC)
```

Comment:
Seems a little odd to create client/host and server/targ aliases instead of just using one.

Reply:
I just did it for legibility, but apparently it had the opposite effect.

Reply:
I think a comment on the uses of whichever pair you keep would be helpful